setup.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+from setuptools import find_packages
+from distutils.core import setup
+import os
+
+this_directory = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(this_directory, 'README.md')) as f:
+    long_description = f.read()
+
+package_name = "SEIR"
+package_version = "0.0.1"
+description = """Hopkins IDD SEIR Simulation and Analysis"""
+
+setup(
+    name=package_name,
+    version=package_version,
+    description=description,
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+
+    author='Joseph Lemaitre',
+    author_email='Jo.lemaitresamra@gmail.com',
+    url='https://github.com/HopkinsIDD/COVIDScenarioPipeline',
+    packages=find_packages(),
+    package_data={},
+    install_requires=[
+        'pandas',
```

Comment:
should specify specific versions -- had several folks ping me with errors and it's been because of version mismatches

Reply:
(assuming the point here is to allow python package outside of docker)

Reply:
It's apparently a weird debate in the python community about what should be in `requirements.txt ` vs. `install_requires`, so I'm not sure: https://packaging.python.org/discussions/install-requires-vs-requirements/

Reply:
the point is to install outside of the docker.  is this resolved?

Reply:
though, the docker image should be a reasonable alternative to needing this (but i don't think i know how to get the docker set up for someone)

Reply:
note if people install this outside of docker (i.e., no standard environment), in for a world of support hurt esp if you don't specify versions (and still even if you do, e.g., you get different results if you run on python 3.8 vs 3.7.6)

Reply:
Right, but if we pin versions, we're in a different world of support hurt b/c folks won't be able to install it at if they have conflicts, right? Can we hard require and/or strongly recommend virtualenvs?

Reply:
Also can we have @jcblemai in here too?

Reply:
i think we should published a blessed docker container and teach folks. it's not that hard to use given the benefit of no more version conflicts

Reply:
I think docker is a mandatory step towards reproducible science, and I really like the idea of having the pipeline in there. That said, adding a layer of learning curve right now, to a pipeline that is already very complex might discourage some users. I think the same for venv, where python people already use them and will debug themselves. I'd vouch right now to be as permissive as possible for packages, that is go the install require route.

I'm curious what happens between python 3.8 vs 3.7.6 ?

Reply:
i guess if this code isn't consistent between python versions, then it doesn't seem reasonable to package it.  I do think that packaging is a goal we want to work towards, but if the current version isn't there i'm inclined not to provide a packaging that will not work.